app/controllers/cheatsheets_controller.rb

```diff
@@ -1,12 +1,26 @@
 class CheatsheetsController < ApplicationController
-  # def new
-  #   @cheatsheet = Cheatsheet.new
-  # end
+  def new
+    @cheatsheet = Cheatsheet.new
+  end
+
+  def show
+    @cheatsheet = Cheatsheet.find(params[:id])
+  end
+
+  def create
+    @cheatsheet = Cheatsheet.new(cheatsheet_params)
+    if @cheatsheet.save
+      flash[:success] = "Welcome to the Learning Together community!"
+      redirect_to @cheatsheet
+    else
+      render 'new'
+    end
+  end
 
   private
 
     def cheatsheet_params
-      params.require(:cheathseet).permit(:title, :topic, :body,
-                                   :author)
+      params.require(:cheathseet).permit(:title, :topic, :content,
```

Comment:
I think it should be `cheatsheet` 🐧 🐶 🐈

Reply:
😆oops, good catch!! 

(tbh I had to stare at that line of code for a while because I couldn't see what was wrong.. is this what coding after work looks like? lol!!)

Reply:
@botanical  ask Laurene about `miletones`.

Reply:
@slabounty lol okay I will 😂